pkgs/development/tools/avro-tools/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchurl, makeWrapper, jre, lib }:
+
+stdenv.mkDerivation rec {
+  version = "1.9.0";
+  name = "avro-tools-${version}";
+
+  src = fetchurl {
+    url =
+    "https://repo1.maven.org/maven2/org/apache/avro/avro-tools/${version}/${name}.jar";
+    sha256 = "164mcz7ljd2ikwsq9ba98galcjar4g4n6ag7kkh466nwrpbmd2zi";
+  };
+
+  phases = "installPhase"; # so nix doesn't attempt to decompress
```

Comment:
We're trying to deprecate explicitly defining phases.
To avoid unpacking you can use `dontUnpack = true;`

Reply:
roger